packages/salesforcedx-vscode/README.md

```diff
@@ -33,7 +33,7 @@ Documentation for each extension as well as additional resources are listed belo
 - [Apex Replay Debugger](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex-replay-debugger)  
    This extension enables VS Code to replay Apex execution from Apex debug logs.
 - [Lighting Component Framework](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-lightning)  
```

Comment:
The link should be changed as well.

Reply:
The link is changing? Will it be
https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-aura
?

Reply:
Sorry, I meant the link text. `[Aura Component Framework]`

Reply:
Oh, right. Yeah, we're also not using "framework" unless we refer to the _whole_ framework (both Aura and LWC). The link text should be just "Aura Components" in this case.